### PR TITLE
Resolves media-query issue and breakpoints

### PR DESCRIPTION
**Pull request by Satira**
Resloves issue #7 
I've added three responsive breakpoints and adjust main menu overlays layout according to them:

- phone-tablet: max-width (899px);

- tablet-landscape: min-width (900px);

- desktop: min-width (1200px);

### CODE REVIEW

File: source/style/modules/sticky-nav.scss [hunk 1]
```diff
@@ -135,115 +147,120 @@ Main Navigation Component
...
-          height: 50px;
-          background: none;
-          border: none;
-          border-bottom: 2px solid $color-bg;
-          cursor: pointer;
-          &> .fas {
-            color: $color-font-primary;
-            font-size: 2.5vh;
-          }
+          font-size: 2vh;
```
Comment (chemicalbr) [inline source/style/modules/sticky-nav.scss]: font-size: 2vh;

**Comment by chemicalbr** [comment]
using vh and vw requires more engagement on the responsive part, will leave it like this for now , probably will need refactor later...

**Comment by chemicalbr** [comment]
using vh and vw requires more engagement on the responsive part, will leave it like this for now , probably will need refactor later...

**Comment by chemicalbr** [comment]
using vh and vw requires more engagement on the responsive part, will leave it like this for now , probably will need refactor later...

File: source/style/modules/sticky-nav.scss (hunk 1 above)
Comment (Satira) [reply source/style/modules/sticky-nav.scss]: Actually I added an issue for that a couple of days ago  - #10 and will update all units in the next Hotfix

**Pull request merged**